### PR TITLE
exam questions done

### PR DESCRIPTION
**Pull request by 7Davronbek**

### CODE REVIEW

File: src/main/java/org/example/GrootMain.java [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
     // todo Sizning vazifangiz main methodi ishga tushunrilgandan keyin har 3 minut kutib har 1 minutda speak methodini ishlatvorish
     //  hint: Thread poollardan foydalanish kerak
-    public static void main( String[] args )
-    {
 
+    public static void main(String[] args) {
+        ScheduledThreadPoolExecutor scheduledThreadPoolExecutor = new ScheduledThreadPoolExecutor(1);
+        scheduledThreadPoolExecutor.scheduleWithFixedDelay(() -> {
+            try {
+                Thread.sleep(3000);
```
Comment (aabdurashidov) [inline src/main/java/org/example/GrootMain.java:15]: kutush uchun thread sleep qilish kere emas edi

File: src/main/java/org/example/MessageService.java [hunk 2]
```diff
@@ -1,15 +1,35 @@
...
-        System.out.printf( "%s xabari %s raqamiga yuborildi%n", message, phoneNumber );
+
+    public static void main(String[] args) throws InterruptedException {
+        sendMessage("Warning", "+998991002030");
+        Lock lock = new ReentrantLock();
+    }
+
+    public static void sendMessage(String message, String phoneNumber) throws InterruptedException {
+        Runnable runnable = () -> {
+            for (int i = 0; i < 3_600_000; i++) {
```
Comment (aabdurashidov) [inline src/main/java/org/example/MessageService.java:20]: nimaga 3.600.000 marta forda aylandiz tushunmadim ))

File: src/main/java/org/example/MessageService.java [hunk 3]
```diff
@@ -1,15 +1,35 @@
...
+                try {
+                    Thread.sleep(10000);
+                } catch (InterruptedException e) {
+                    throw new RuntimeException(e);
+                }
+                System.out.printf("%s xabari %s raqamiga yuborildi%n", message, phoneNumber);
+            }
+        };
+
+        for (int i = 0; i < 10; i++) {
```
Comment (aabdurashidov) [inline src/main/java/org/example/MessageService.java:30]: bu yerda nimaga 10 marta for?

File: src/main/java/org/example/MessageService.java (hunk 2 above)
Comment (7Davronbek) [reply src/main/java/org/example/MessageService.java:20]: Bo'tta manimcha sms jonatish keregidi, 10 martada 3.600.000 ta thread yaratib sms jonatmoqchidim shekili)

**Pull request closed without merging**